Code/Framework/AzToolsFramework/AzToolsFramework/Entity/PrefabEditorEntityOwnershipService.h

```diff
@@ -215,5 +216,6 @@ namespace AzToolsFramework
         Prefab::PrefabLoaderInterface* m_loaderInterface;
         AzFramework::EntityContextId m_entityContextId;
         AZ::SerializeContext m_serializeContext;
+        bool m_isLevelLoaded = false;
```

Comment:
I don't remember why we constructed m_rootInstance in Initialize() method. But if we can delay it until the first time user creates a level, we can just use 'IsInitialized()' instead of relying on this bool right? Can you see if that's a possibility?

Reply:
I have tried that, rootInstance is not null when no level is loaded. It has a "NewLevel" instance assigned.